symfony/src/Form/MeasurementType.php

```diff
@@ -94,11 +94,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             } elseif ($type === 'date') {
                 unset($fieldOptions['scale']);
                 $class = Type\DateType::class;
+                $minDate = new \DateTime('-6 months');
```

Comment:
For clarity, can you modify $minDate here and then use in the constraint below, instead of modifying it in the same line where it is assigned?